extension/src/experiments/columns/model.ts

```diff
@@ -82,6 +84,13 @@ export class ColumnsModel extends PathSelectionModel<Column> {
     return this.data.length > 1
   }
 
+  private getFirstThreeColumnOrderFromData(): string[] {
+    return this.data
```

Comment:
This seemed like the easiest way to collect the missing `columnOrder` but is there a better way to fix this bug?

Reply:
Maybe we should just add a default `columnOrderState` if it's empty instead. We could do that on line 112. That way, we'll never have to check whether it's empty or not.

Reply:
Good idea! From a little testing I did, it will take a bit of time to make that change so I'll update in a follow-up at a later date!